### PR TITLE
[Imath] build with gcc5

### PR DESCRIPTION
**Pull request by twadleigh**
Build with gcc5. As it is, openexr support can't be built into ImageMagick_jll.

### CODE REVIEW

File: I/Imath/build_tarballs.jl [hunk 1]
```diff
@@ -36,4 +36,4 @@ dependencies = Dependency[
 ]
 
 # Build the tarballs, and possibly a `build.jl` as well.
-build_tarballs(ARGS, name, version, sources, script, platforms, products, dependencies; preferred_gcc_version = v"7.1.0")
+build_tarballs(ARGS, name, version, sources, script, platforms, products, dependencies)
```
Comment (giordano) [inline I/Imath/build_tarballs.jl:39]: I don't think gcc v4.8, the default toolchain, has full support for c++11, you need at least version 5 for that

Comment (twadleigh) [reply I/Imath/build_tarballs.jl:39]: BTW, I'm having trouble being able to get `BinaryBuilder` to do its thing pointing to locally built, not yet published, artifacts. The issue I'm trying to work through involves touching three build recipes. I'm sure it's in the docs somewhere, but I haven't found the right passage, or, if I have, I haven't executed the advice correctly. @giordano , would you be able to point me in the right direction?

Comment (giordano) [reply I/Imath/build_tarballs.jl:39]: https://docs.binarybuilder.org/dev/FAQ/#Can-I-publish-a-JLL-package-locally-without-going-through-Yggdrasil?-1 Note that the documentation mentions using repositories on github as source, specified in PackageSpec, but in practice you can use whatever PackageSpec allows you to, including a local directory (so you don't have to deploy the packages remotely, can do everything locally)

Comment (twadleigh) [reply I/Imath/build_tarballs.jl:39]: OK, I'm re-reading [this](https://docs.binarybuilder.org/stable/jll/#Overriding-the-artifacts-in-JLL-packages) more carefully. I think I missed some subtleties last night.

**Comment by twadleigh** [comment]
Related: looks like I probably shouldn't have registered my own [Imath_jll](https://github.com/twadleigh/Imath_jll.jl). Should I transfer ownership? Or, maybe better, just delete it and have a new one created and registered?

**Comment by giordano** [comment]
> Related: looks like I probably shouldn't have registered my own Imath_jll.

I'm not sure what you're talking about

**Review by giordano — APPROVED** [review]
Looks good to me, just waiting to clarify the bit about registration

**Comment by twadleigh** [comment]
> Looks good to me, just waiting to clarify the bit about registration

Nevermind. I was under the impression that `twadleigh/Imath_jll.jl` was the registered version. I looked closer and saw that the JuliaBinaryWrappers version is.

**Pull request merged**